jackson-databind/src/main/java/io/micronaut/jackson/modules/BeanIntrospectionModule.java

```diff
@@ -270,19 +304,7 @@ public BeanSerializerBuilder updateBuilder(SerializationConfig config, BeanDescr
                         if (beanProperty.hasAnnotation(JsonIgnore.class)) {
                             continue;
                         }
-                        final String propertyName;
```

Comment:
I assume we don't need this due to : 

https://github.com/micronaut-projects/micronaut-core/blob/3.4.x/http/src/main/java/io/micronaut/http/hateoas/Resource.java#L54

and: 

https://github.com/micronaut-projects/micronaut-core/blob/3.4.x/http/src/main/java/io/micronaut/http/hateoas/Resource.java#L46

Reply:
We have a test wchih verifies this works: 

`BeanIntrospectionModuleSpec::"void "test that introspected serialization of errors works"`